diesel/src/query_builder/debug_query.rs

```diff
@@ -81,6 +81,8 @@ where
     DB: Backend,
     T: QueryFragment<DB>,
 {
+    // Clippy is wrong, this cannot be expressed with pointer casting
```

Comment:
Is this worth reporting a bug on clippy ?

Reply:
I did